service-front/module/Application/src/Helpers/FormProcessorHelper.php

```diff
@@ -246,7 +246,9 @@ public function processTemplate(array $fraudCheck, array $templates): mixed
                 $template = $templates['thin_file'];
                 break;
             case 'STOP':
-                throw new OpgApiException('Stop response received from fraud check service.');
+                $template = $templates['fraud'];
+//                throw new OpgApiException('Stop response received from fraud check service.');
```

Comment:
Presume this was commented out as we will handle a STOP with the 4/4 questions, but best to just remove if that's the case to avoid confusion.